README.md

```diff
@@ -1,8 +1,150 @@
+```bash
+git clone git@github.com:scroll-tech/scroll-sdk.git
+cd scroll-sdk/devnet
+```
+
+This next command manually pulls and extracts the charts for the latest version of Scroll SDK.
+```bash
+make bootstrap
+```
+> ⚠️ **Encountering Permission Issues?**<br>
+> If you are given any prompts during the execution of the bootstrap command or if `make install` does not successfully execute, then there is likely a root vs. user mismatch between configuration shell scripts and the downloaded files.
+> You can solve this by exiting the bootstrap process with `ctrl+c` and running the `config` command separately with `sudo make config`.
+
+
+### Installing the Helm Chart
+
+In this step, we will launch the sequencing layer, the coordinator, and various visibility services.
+Because provers require manual configuration, we will start those separately.
+Note that the standard devnet settings do not include any proving layer services, so you will need to add the following two lines to the end of the `install` command in `scroll-sdk/devnet/Makefile`.
+```bash
+    --set coordinator-api.enabled=true \
+    --set coordinator-cron.enabled=true
+```
+After that adjustment, enter `make install` in your terminal to start the chain.
+You can monitor the progress and health of all the services via the command `kubectl get pods`.
+
+> 🕒 **Waiting on the coordinator?**<br>
+> An essential functionality of the coordinator is the validation of proofs that are returned from a prover.
+> For this reason, there are two initial pods (`assets-download` and `parameter-download`) which must pull zkEVM circuit artifact files before the final `coordinator-api` pod starts.
+> You can watch the progress of the first pod with `kubectl logs coordinator-api-0 -c assets-download`.
+> Similarly, you can watch the progress of the trusted setup download via `kubectl logs coordinator-api-0 -c parameter-download`.
+
+### Accessing Services Locally
+
+Now, you should add mappings for the Scroll SDK services to your local DNS resolver by appending the following lines to the end of your `/etc/hosts` file:
+```bash
+127.0.0.1 frontends.scrollsdk
+127.0.0.1 blockscout.scrollsdk
+127.0.0.1 bridge-history-api.scrollsdk
+127.0.0.1 grafana.scrollsdk
+127.0.0.1 l1-devnet.scrollsdk
+127.0.0.1 l1-explorer.scrollsdk
+127.0.0.1 l2-rpc.scrollsdk
+127.0.0.1 rollup-explorer-backend.scrollsdk
+127.0.0.1 coordinator-api.scrollsdk
+```
+
+You should be able to access the endpoints via your browser.
+[This section](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/#web-uis) of the Scroll SDK documentation provides an explanation of all the available dashboards.
+
+### 🚀 Launching Sindri Provers
+
+The simplest way to deploy Sindri provers is by following the guide in our [demo directory](demo/README.md).
+Before proceeding to that step, you may find it useful to generate some initial traffic on your chain so that the provers can be sent tasks.
```

Comment:
I would add clarification that this should be run from `scroll-sdk/devnet/scroll-sdk` -- otherwise the command won't execute.